script.js

```diff
@@ -0,0 +1,12 @@
+isThisMyName('Adam');
+
+
+function isThisMyName(firstName) {
+    firstNameLowerCase = firstName.toLowerCase()
+    if (firstNameLowerCase === "natasza") {
+        result = true;
+    } else {
+        result = false;
+    }
```

Comment:
[this](https://github.com/nattsnb/js-basics/pull/2#discussion_r1355029126) comment is still relevant please try this approach:
```
function isLetterC(letter) {
  return letter === "C";
}
```